Demo.ipynb

```diff
@@ -12,14 +11,12 @@
     "import math\n",
     "import numpy as np\n",
     "import matplotlib.pyplot as plt\n",
-    "import pv_model_210131 as pv\n",
-    "import pv_model_class\n"
+    "import pv_model_class"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 2,
-   "id": "little-jungle",
```

Comment:
Yeah that is interesting. I'm not sure why the ID would have been removed. Perhaps because my Python or my packages are out of date it didn't know how to handle the IDs? It appears that JEP is an acronym for [Jupyter Enhancement Proposal](https://jupyter.org/enhancement-proposals/29-jep-process/jep-process.html).